### DISCUSSION TITLE
Only GET API:s

### DISCUSSION THREAD

**Discussion by RikardEdgren001** [EMResearch/EvoMaster#1100] [Q&A]
What is the best way to only run EvoMaster on GET APIs? (I don't want to mess up anything in a test environment)
For example on https://petstore.swagger.io/

**Comment by arcuri82** [comment]
Hi,
  there is no direct support for this. However, it could be achieved indirectly by using `--endpointTagFilter`: `Comma-separated list of OpenAPI/Swagger 'tags' definitions. Only the REST endpoints having at least one of such tags will be fuzzed. If no tag is specified here, then such filter is not applied.`
